docs/source/reference/hit.rst

```diff
@@ -0,0 +1,16 @@
+.. _hit:
+
+HIT (High-energy Ion Telescope)
+==========================
+
+.. currentmodule:: imap_processing.hit.l0
```

Comment:
```suggestion
.. currentmodule:: imap_processing.hit
```
Do we want to make this be the `hit` base module, and then automodule/autosummary it so that it picks up your future levels as well?

Or I think you could put in l0.hit_l1a_decom which might be more explicit here that you're in the l0 code path as well.

Reply:
` l0.hit_l1a_decom` worked